curation_concerns-models/app/models/concerns/curation_concerns/file/virus_check.rb

```diff
@@ -10,9 +10,9 @@ module VirusCheck
       # Default behavior is to raise a validation error and halt the save if a virus is found
       def detect_viruses
         return unless content.changed?
-        Sufia::GenericFile::Actor.virus_check(local_path_for_content)
+        CurationConcerns::GenericFile::Actor.virus_check(local_path_for_content)
```

Comment:
Use `CurationConcerns::GenericFileActor`, not `CurationConcerns::GenericFile::Actor`